build.sbt

```diff
@@ -648,7 +696,7 @@ lazy val mtest = project
       scalaVersion.value,
     ),
   )
-  .dependsOn(mtags)
+  .dependsOn(mtags % "compile->test")
```

Comment:
Because we add `scala` only as test dependencies, we have to specify that `mtest` compilation depends on `Test` of mtags3WithPresentationCompiler.
```scala
  Test / unmanagedSourceDirectories ++= {
    val base = (Compile / sourceDirectory).value
    if (isScala3PresentationCompiler(scalaVersion.value)) {
      List(base / s"scala")
    } else {
      Nil
    }
  },
```

Reply:
Maybe do it here:
```
Test/ unmanagedSourceDirectories ++= {
      mtags.base  / s"scala-3-wrapper"
    }
```
will that work?